coral-spark/src/main/java/com/linkedin/coral/spark/IRRelToSparkRelTransformer.java

```diff
@@ -323,6 +323,7 @@ public RexNode visitCall(RexCall call) {
       return Optional.empty();
     }
 
+    // TODO:This removal is no longer needed
```

Comment:
Can we handle it in this PR and modify the corresponding tests in Coral-Spark?
`testUnionExtractUDF` in `CoralSparkTest` will fail if we don't modify it.

Reply:
And before committing, we can run `./gradlew clean build` to make sure that the Github CI can pass.

Reply:
thx for the suggestion. will address.

Reply:
actually what about temporarily disabling the `testUnionExtractUDF` as part of this PR ? I do want to separate them out since change in this PR is happening in the Coral IR that could have broader impact.

Reply:
Sure, I am fine with separating them.